src/NetEscapades.AspNetCore.SecurityHeaders/HeaderPolicyCollectionExtensions.cs

```diff
@@ -80,6 +83,12 @@ public static HeaderPolicyCollection AddDefaultApiSecurityHeaders(this HeaderPol
         // The following are generally not applicable, but still worth applying for safety
         policies.AddReferrerPolicyNoReferrer();
         policies.AddPermissionsPolicyWithDefaultSecureDirectives();
+
+        policies.AddReferrerPolicyStrictOriginWhenCrossOrigin();
+        policies.AddCrossOriginOpenerPolicy(builder => builder.SameOrigin());
+        policies.AddCrossOriginEmbedderPolicy(builder => builder.RequireCorp());
+        policies.AddCrossOriginResourcePolicy(builder => builder.SameOrigin());
```

Comment:
This one I'm unsure about, it could be very confusing to people I think 🤔 It basically "breaks" CORS as I understand it? i.e. if you set this, then even if you have enabled CORS requests using `Access-Control-Allow-Origin` then a _bunch_ of clients are going to break. I'm torn on whether we should enable that by default or not tbh 🤔  _Maybe_ `SameSite()` is a better, more practical option? 

The thing is, I'm not necessarily trying to have these headers be the absolute maxed-out most secure, they're meant to be a good _practical_ base, that you can throw into your app, and it will _probably_ be ok... My concern is that this one locks down your API _too_ much by default...

Reply:
Switched to SameSite, makes sense